aspnetcore/migration/mvc.md

```diff
@@ -3,18 +3,189 @@ title: Migrate from ASP.NET MVC to ASP.NET Core MVC
+
+1. Copy the methods from the ASP.NET MVC `HomeController` to replace the new ASP.NET Core `HomeController` methods. There's no need to change the return type of the action methods. The ASP.NET MVC built-in template's controller action method return type is [ActionResult](https://msdn.microsoft.com/library/system.web.mvc.actionresult(v=vs.118).aspx); in ASP.NET Core MVC, the action methods return `IActionResult` instead. `ActionResult` implements `IActionResult`.
+1. In the ASP.NET Core project, right-click the **Views** > **Home** folder, select  **Add** > **Existing Item...**.
+1. In the **Add Existing Item dialog**, navigate to the ASP.NET MVC WebApp1 project > Views > Home directory.
+1. Select the *About.cshtml*, *Contact.cshtml*, and *Index.cshtml* Razor view files, then select **Add**, replacing the existing files.
+1. Run the ASP.NET Core app.
+
+See [Controllers](xref:mvc/controllers/actions) and [Views](xref:mvc/views/overview) for more information.
+
+## Test each method
+
+The layout file and styles have not been migrated yet, so the rendered views only contain the content in the view files.  Also, the layout file generated links for the `About` and `Contact` views will not be available yet.
+
+* Invoke the rendered views from the browser on the running ASP.NET core app by replacing the current port number with the port number used in the ASP.NET core project. For example: `https://localhost:44375/home/about`.
+
+See [ASP.NET Core Controllers](xref:mvc/controllers/actions) and [ASP.NET Core Views](xref:mvc/views/overview) for more information.
+
+## Migrate Static content
+
+In previous versions of ASP.NET MVC, static content was hosted from the root of the web project and was intermixed with server-side files. In ASP.NET Core, static content is hosted in the **wwwroot** folder.
+
+Copy the static content from the ASP.NET MVC *WebApp1* project to the *wwwroot* folder in the ASP.NET Core *WebApp1* project:
+
+1. In the ASP.NET Core project, right-click the **wwwroot** folder, select  **Add** > **Existing Item...**.
+1. In the **Add Existing Item dialog**, navigate to the ASP.NET MVC *WebApp1* project.
+1. Select the *favicon.ico* file, then select **Add**, replacing the existing file.
+
+## Migrate the layout files
+
+Copy the ASP.NET MVC project layout files to the ASP.NET Core project:
+
+1. In the ASP.NET Core project, right-click the **Views** folder, select  **Add** > **Existing Item...**.
+1. In the **Add Existing Item dialog**, navigate to the ASP.NET MVC *WebApp1* project > *Views* directory.
+1. Select  *_ViewStart.cshtml* file then select **Add**.
+
+Copy the ASP.NET MVC project shared layout files to the ASP.NET Core project:
+
+1. In the ASP.NET Core project, right-click the **Views** > **Shared** folder, select  **Add** > **Existing Item...**.
+1. In the **Add Existing Item dialog**, navigate to the ASP.NET MVC *WebApp1* project > *Views* > *Shared* directory.
```

Comment:
```suggestion
1. In the **Add Existing Item** dialog, navigate to the ASP.NET MVC *WebApp1* project > *Views* > *Shared* directory.
```

Reply:
> "In the ASP.NET Core project, right-click the wwwroot folder, select Add > Existing Item....

Make sure none of those get localized. Look at localized versions.

Reply:
@Rick-Anderson, good catch. This text is new to this update, so I didn't have a previous localized version to verify, however, I checked around in our existing content, an **Add** > and other similar right click instruction is getting localized, a wide spread problem:  

Example:
https://github.com/dotnet/AspNetCore.Docs/blob/live/aspnetcore/includes/scaffold-identity/id-scaffold-dlg.md
Compared to:
https://github.com/dotnet/AspNetCore.Docs.zh-cn/blob/live/aspnetcore/includes/scaffold-identity/id-scaffold-dlg.md

For this one I will go back and add no loc fencing and create a new issue for the repo wide problem.

Reply:
Or just code fence it.